coala_quickstart/generation/SettingsFilling.py

```diff
@@ -210,8 +213,20 @@ def require_setting(setting_name, setting_info, section):
     user_input = input()
 
     if setting_info["type"]:
+
         try:
-            user_input = setting_info["type"](user_input)
+            if setting_info["type"] is bool:
+                processed_input = " ".join(re.findall("[a-zA-Z0-9\-]+",
+                                                      user_input.lower()))
```

Comment:
do we need this? If the user enters an incorrect value, just tell them and ask them to reenter

assuming silently is usually not a great idea :P